obs/transform.py

```diff
@@ -19,16 +18,13 @@ class TransformOBS(object):
 
     dir_re = re.compile(r'(<div\s.*?class=".*?obs-content.*?">).*?(</div>)', re.UNICODE + re.DOTALL)
```

Comment:
If I had the html string ``<div class="a" obs-content="b"><div class="another"></div></div>``, this regular expression would match ``<div class="a" obs-content="b"><div class="another"></div>``, and not ``<div class="a" obs-content="b"><div class="another"></div></div>``. In other words, it doesn't ensure that the closing ``</div>`` aligns with the opening ``<div .. >``.

(P.S. If this behavior is in fact intentional, then please disregard this comment.)